From 4058eb4f688d02b5ed3ff3d4297042c7f0b1c786 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:26:11 +0100
Subject: [PATCH 03/39] core: use mmap to load files.

---
 src/core/load-fragment.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/load-fragment.c b/src/core/load-fragment.c
index 9d5c39b3d..a770fdb3f 100644
--- a/src/core/load-fragment.c
+++ b/src/core/load-fragment.c
@@ -4237,7 +4237,7 @@ static int open_follow(char **filename, FILE **_f, Set *names, char **_final) {
                 *filename = target;
         }
 
-        f = fdopen(fd, "re");
+        f = fdopen(fd, "rem");
         if (!f) {
                 safe_close(fd);
                 return -errno;
-- 
2.13.3

