From def1a3ba45e81bba3025e3a5fd71c031a28e9063 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:25:41 +0100
Subject: [PATCH 03/44] journal: clearout & drop kmsg.

---
 src/journal/journald-kmsg.c | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/src/journal/journald-kmsg.c b/src/journal/journald-kmsg.c
index ce82102eed..b7546129ff 100644
--- a/src/journal/journald-kmsg.c
+++ b/src/journal/journald-kmsg.c
@@ -366,6 +366,7 @@ static int dispatch_dev_kmsg(sd_event_source *es, int fd, uint32_t revents, void
 int server_open_dev_kmsg(Server *s) {
         mode_t mode;
         int r;
+        char buffer[40960];
 
         assert(s);
 
@@ -384,6 +385,14 @@ int server_open_dev_kmsg(Server *s) {
         if (!s->read_kmsg)
                 return 0;
 
+        /* clear out /dev/kmsg, we don't want all its messages */
+        while (1) {
+                int ret;
+                ret = read(s->dev_kmsg_fd, buffer, 40960);
+                if (ret <= 0)
+                        break;
+        }
+
         r = sd_event_add_io(s->event, &s->dev_kmsg_event_source, s->dev_kmsg_fd, EPOLLIN, dispatch_dev_kmsg, s);
         if (r < 0) {
 
-- 
2.21.0

