From 9cdb3cda5981fea693427435d2da5c796860cb5b Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:36:29 +0100
Subject: [PATCH 04/28] Makefile.am: drop pam/nsswitch, ship legacy tmpfiles.

---
 Makefile.am | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 3b7cc1e..9c01d7b 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -2389,10 +2389,8 @@ dist_tmpfiles_DATA = \
 	tmpfiles.d/systemd-nspawn.conf \
 	tmpfiles.d/journal-nocow.conf
 
-if HAVE_SYSV_COMPAT
 dist_tmpfiles_DATA += \
 	tmpfiles.d/legacy.conf
-endif
 
 SYSINIT_TARGET_WANTS += \
 	systemd-tmpfiles-setup-dev.service \
@@ -2453,16 +2451,6 @@ EXTRA_DIST += \
 	sysusers.d/basic.conf.in
 
 # ------------------------------------------------------------------------------
-dist_factory_etc_DATA = \
-	factory/etc/nsswitch.conf
-
-if HAVE_PAM
-dist_factory_pam_DATA = \
-	factory/etc/pam.d/system-auth \
-	factory/etc/pam.d/other
-endif
-
-# ------------------------------------------------------------------------------
 if ENABLE_FIRSTBOOT
 systemd_firstboot_SOURCES = \
 	src/firstboot/firstboot.c
-- 
2.9.0

