From 3f5c6f6db67cf0ca2245b35613f5a94404ae04b8 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:36:29 +0100
Subject: [PATCH 04/39] Makefile.am: drop pam/nsswitch, ship legacy tmpfiles.

---
 Makefile.am | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index c16e62280..abcf41ec2 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -2697,10 +2697,8 @@ dist_tmpfiles_DATA = \
 	tmpfiles.d/systemd-nspawn.conf \
 	tmpfiles.d/journal-nocow.conf
 
-if HAVE_SYSV_COMPAT
 dist_tmpfiles_DATA += \
 	tmpfiles.d/legacy.conf
-endif
 
 if HAVE_REMOTE
 nodist_tmpfiles_DATA += \
@@ -2768,16 +2766,6 @@ EXTRA_DIST += \
 	sysusers.d/basic.conf.in
 
 # ------------------------------------------------------------------------------
-dist_factory_etc_DATA = \
-	factory/etc/nsswitch.conf
-
-if HAVE_PAM
-dist_factory_pam_DATA = \
-	factory/etc/pam.d/system-auth \
-	factory/etc/pam.d/other
-endif
-
-# ------------------------------------------------------------------------------
 if ENABLE_FIRSTBOOT
 systemd_firstboot_SOURCES = \
 	src/firstboot/firstboot.c
-- 
2.13.3

