From 413cb7a591d04269aae406be2c0f8ee416832d72 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:26:11 +0100
Subject: [PATCH] core: use mmap to load files.

---
 src/core/load-fragment.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/load-fragment.c b/src/core/load-fragment.c
index 73cc8adc2..151b526a0 100644
--- a/src/core/load-fragment.c
+++ b/src/core/load-fragment.c
@@ -4175,7 +4175,7 @@ static int open_follow(char **filename, FILE **_f, Set *names, char **_final) {
                 free_and_replace(*filename, target);
         }
 
-        f = fdopen(fd, "re");
+        f = fdopen(fd, "rem");
         if (!f) {
                 safe_close(fd);
                 return -errno;
-- 
2.19.0

