From 8ce4c2b21ca3082bee2047df7d8718c6893dae05 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:38:31 +0100
Subject: [PATCH 05/44] journal: flush var/kmsg after starting.

---
 src/journal/journald.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/journal/journald.c b/src/journal/journald.c
index 9a7cb3e1fc..019952fd2a 100644
--- a/src/journal/journald.c
+++ b/src/journal/journald.c
@@ -40,8 +40,6 @@ int main(int argc, char *argv[]) {
                 goto finish;
 
         server_vacuum(&server, false);
-        server_flush_to_var(&server, true);
-        server_flush_dev_kmsg(&server);
 
         if (server.namespace)
                 log_debug("systemd-journald running as PID "PID_FMT" for namespace '%s'.", getpid_cached(), server.namespace);
@@ -53,6 +51,9 @@ int main(int argc, char *argv[]) {
                               LOG_MESSAGE("Journal started"),
                               NULL);
 
+        server_flush_to_var(&server, true);
+        server_flush_dev_kmsg(&server);
+
         /* Make sure to send the usage message *after* flushing the
          * journal so entries from the runtime journals are ordered
          * before this message. See #4190 for some details. */
-- 
2.26.0

