From b6f2ca999f96f0ba343aa2572015beef34fe4c40 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:38:31 +0100
Subject: [PATCH 07/38] journal: flush var/kmsg after starting.

---
 src/journal/journald.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/journal/journald.c b/src/journal/journald.c
index 5e7b1dcb4..e7122ce6b 100644
--- a/src/journal/journald.c
+++ b/src/journal/journald.c
@@ -37,8 +37,6 @@ int main(int argc, char *argv[]) {
                 goto finish;
 
         server_vacuum(&server, false);
-        server_flush_to_var(&server, true);
-        server_flush_dev_kmsg(&server);
 
         log_debug("systemd-journald running as pid "PID_FMT, getpid_cached());
         server_driver_message(&server, 0,
@@ -46,6 +44,9 @@ int main(int argc, char *argv[]) {
                               LOG_MESSAGE("Journal started"),
                               NULL);
 
+        server_flush_to_var(&server, true);
+        server_flush_dev_kmsg(&server);
+
         /* Make sure to send the usage message *after* flushing the
          * journal so entries from the runtime journals are ordered
          * before this message. See #4190 for some details. */
-- 
2.17.1

