From 9f9ce56c067c24682d6377e14caeb4685e46395d Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:37:04 +0100
Subject: [PATCH 07/30] bootchart: mount /proc early.

---
 src/bootchart/bootchart.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/bootchart/bootchart.c b/src/bootchart/bootchart.c
index 6a0e1d6..b01c25f 100644
--- a/src/bootchart/bootchart.c
+++ b/src/bootchart/bootchart.c
@@ -42,6 +42,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <sys/mount.h>
 #include <sys/resource.h>
 #include <time.h>
 #include <unistd.h>
@@ -378,6 +379,8 @@ int main(int argc, char *argv[]) {
                 graph_start = log_start - uptime;
         }
 
+	mount("/proc", "/proc", "proc", 0, NULL);
+
         if (graph_start < 0.0) {
                 log_error("Failed to setup graph start time.\n\n"
                           "The system uptime probably includes time that the system was suspended. "
-- 
2.8.1

