From 54cabda7a96076094d627c5bb4a57902aa29fff9 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:38:31 +0100
Subject: [PATCH 08/30] journal: flush var/kmsg after starting.

---
 src/journal/journald.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/journal/journald.c b/src/journal/journald.c
index 272acb7..f205117 100644
--- a/src/journal/journald.c
+++ b/src/journal/journald.c
@@ -52,14 +52,15 @@ int main(int argc, char *argv[]) {
                 goto finish;
 
         server_vacuum(&server, false, false);
-        server_flush_to_var(&server);
-        server_flush_dev_kmsg(&server);
 
         log_debug("systemd-journald running as pid "PID_FMT, getpid());
         server_driver_message(&server, SD_MESSAGE_JOURNAL_START,
                               LOG_MESSAGE("Journal started"),
                               NULL);
 
+        server_flush_to_var(&server);
+        server_flush_dev_kmsg(&server);
+
         for (;;) {
                 usec_t t = USEC_INFINITY, n;
 
-- 
2.8.2

