From c01d92d95e126faf6aa75a4ac6331e5c016cddd6 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:40:24 +0100
Subject: [PATCH 09/39] logind,pam: fix systemd-user to include common-session.

---
 factory/etc/pam.d/other | 1 +
 1 file changed, 1 insertion(+)

diff --git a/factory/etc/pam.d/other b/factory/etc/pam.d/other
index c1f729659..83408c327 100644
--- a/factory/etc/pam.d/other
+++ b/factory/etc/pam.d/other
@@ -4,3 +4,4 @@ auth     include system-auth
 account  include system-auth
 password include system-auth
 session  include system-auth
+session  include common-session
-- 
2.17.1

