From a78b18f0a8ba69596285d3b21e0d2f6614b252a1 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:41:36 +0100
Subject: [PATCH 09/32] configure.ac: disable pie.

---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 3835413ad..12c07d9d8 100644
--- a/configure.ac
+++ b/configure.ac
@@ -238,7 +238,6 @@ CC_CHECK_FLAGS_APPEND([with_ldflags], [LDFLAGS], [\
         -Wl,--no-undefined \
         -Wl,-z,relro \
         -Wl,-z,now \
-        -pie \
         -Wl,-fuse-ld=gold])
 
 AC_SUBST([OUR_CPPFLAGS], "$with_cppflags $sanitizer_cppflags")
-- 
2.12.2

