From e925478a1944e42fceb64cba37b24103976c47ee Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:50:13 +0100
Subject: [PATCH 09/44] core: do not apply presets.

---
 src/core/manager.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/core/manager.c b/src/core/manager.c
index 25afdbea04..71d7e9f03b 100644
--- a/src/core/manager.c
+++ b/src/core/manager.c
@@ -1658,8 +1658,6 @@ int manager_startup(Manager *m, FILE *serialization, FDSet *fds) {
         if (r < 0)
                 return r;
 
-        manager_preset_all(m);
-
         lookup_paths_log(&m->lookup_paths);
 
         {
-- 
2.26.0

