From ca84025a419f61e5b2e9aee58f3381972fcc977d Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:50:13 +0100
Subject: [PATCH 09/42] core: do not apply presets.

---
 src/core/manager.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/core/manager.c b/src/core/manager.c
index 8884437347..9a7326fb6c 100644
--- a/src/core/manager.c
+++ b/src/core/manager.c
@@ -1721,8 +1721,6 @@ int manager_startup(Manager *m, FILE *serialization, FDSet *fds) {
         if (r < 0)
                 return r;
 
-        manager_preset_all(m);
-
         lookup_paths_log(&m->lookup_paths);
 
         {
-- 
2.32.0

