From ebd82e5b224b5b9a708fd15b7af1a5f3e495f269 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:50:44 +0100
Subject: [PATCH 10/45] locale-setup: set default locale to a unicode one.

---
 src/core/locale-setup.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/core/locale-setup.c b/src/core/locale-setup.c
index 33728b0ff2..3af4ff6cc0 100644
--- a/src/core/locale-setup.c
+++ b/src/core/locale-setup.c
@@ -58,6 +58,12 @@ int locale_setup(char ***environment) {
                         log_warning_errno(r, "Failed to read /etc/locale.conf: %m");
         }
 
+        /* Nothing set... use some UTF-8 locale */
+        if (r <= 0) {
+                variables[VARIABLE_LANG] = strdup("en_US.UTF-8");
+        }
+
+        add = NULL;
         for (i = 0; i < _VARIABLE_LC_MAX; i++) {
                 char *s;
 
-- 
2.26.0

