From ec182983b9d712d9b5cae5e5aef1a7c3f0ac25f0 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:50:44 +0100
Subject: [PATCH 10/41] locale-setup: set default locale to a unicode one.

---
 src/core/locale-setup.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/core/locale-setup.c b/src/core/locale-setup.c
index aa4a89c17a..186fa12f05 100644
--- a/src/core/locale-setup.c
+++ b/src/core/locale-setup.c
@@ -59,6 +59,12 @@ int locale_setup(char ***environment) {
                         log_warning_errno(r, "Failed to read /etc/locale.conf: %m");
         }
 
+        /* Nothing set... use some UTF-8 locale */
+        if (r <= 0) {
+                variables[VARIABLE_LANG] = strdup("en_US.UTF-8");
+        }
+
+        add = NULL;
         for (i = 0; i < _VARIABLE_LC_MAX; i++) {
                 char *s;
 
-- 
2.23.0

