From ceb3b8a49d78ed68849477e3d72e473cbfdefecf Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Mon, 24 Aug 2015 14:42:53 +0100
Subject: [PATCH 11/38] Ship default services in system unit dir.

---
 units/meson.build | 23 +++++++++++++++--------
 1 file changed, 15 insertions(+), 8 deletions(-)

--- systemd-stable-255.2/units/meson.build.org	2024-01-02 15:09:30.290857163 +0000
+++ systemd-stable-255.2/units/meson.build	2024-01-02 15:16:59.664937452 +0000
@@ -178,7 +178,9 @@
           'symlinks' : ['initrd-root-device.target.wants/'],
         },
         { 'file' : 'remote-fs-pre.target' },
-        { 'file' : 'remote-fs.target' },
+        { 'file' : 'remote-fs.target',
+          'symlinks' : ['multi-user.target.wants/'],
+	},
         {
           'file' : 'remote-veritysetup.target',
           'conditions' : ['HAVE_LIBCRYPTSETUP'],
@@ -408,6 +410,7 @@
         {
           'file' : 'systemd-networkd-wait-online.service.in',
           'conditions' : ['ENABLE_NETWORKD'],
+          'symlinks' : ['network-online.target.wants/'],
         },
         {
           'file' : 'systemd-networkd-wait-online@.service.in',
@@ -416,6 +419,7 @@
         {
           'file' : 'systemd-networkd.service.in',
           'conditions' : ['ENABLE_NETWORKD'],
+          'symlinks' : ['multi-user.target.wants/', 'dbus-org.freedesktop.network1.service'],
         },
         {
           'file' : 'systemd-networkd.socket',
@@ -525,7 +529,9 @@
           'symlinks' : ['sysinit.target.wants/'],
         },
         { 'file' : 'systemd-reboot.service' },
-        { 'file' : 'systemd-remount-fs.service.in' },
+        { 'file' : 'systemd-remount-fs.service.in',
+	  'symlinks' : ['local-fs.target.wants/'],
+	},
         {
           'file' : 'systemd-repart.service.in',
           'conditions' : ['ENABLE_REPART'],
@@ -534,6 +540,7 @@
         {
           'file' : 'systemd-resolved.service.in',
           'conditions' : ['ENABLE_RESOLVE'],
+	  'symlinks' : ['multi-user.target.wants/', 'dbus-org.freedesktop.resolve1.service'],
         },
         {
           'file' : 'systemd-rfkill.service.in',
@@ -607,6 +614,8 @@
         {
           'file' : 'systemd-timesyncd.service.in',
           'conditions' : ['ENABLE_TIMESYNCD'],
+	  'symlinks' : ['multi-user.target.wants/'],
+
         },
         {
           'file' : 'systemd-tmpfiles-clean.service',
