From 9b8698fcca07e146249240058f4be1e1aebdc272 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:50:13 +0100
Subject: [PATCH 11/44] core: do not apply presets.

---
 src/core/manager.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/core/manager.c b/src/core/manager.c
index 12ae911a38..b8e056e3ea 100644
--- a/src/core/manager.c
+++ b/src/core/manager.c
@@ -1655,12 +1655,11 @@ int manager_startup(Manager *m, FILE *serialization, FDSet *fds) {
         if (r < 0)
                 return r;
 
-        manager_preset_all(m);
-
         r = lookup_paths_reduce(&m->lookup_paths);
         if (r < 0)
                 log_warning_errno(r, "Failed ot reduce unit file paths, ignoring: %m");
 
+        lookup_paths_reduce(&m->lookup_paths);
         manager_build_unit_path_cache(m);
 
         {
-- 
2.21.0

