From 0766f3a8d9a4a1a7a91668f830a47f0b46a32c4b Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:50:13 +0100
Subject: [PATCH 13/39] core: do not apply presets.

---
 src/core/manager.c | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/src/core/manager.c b/src/core/manager.c
index 84adb9c66..23b75d750 100644
--- a/src/core/manager.c
+++ b/src/core/manager.c
@@ -1432,19 +1432,6 @@ int manager_startup(Manager *m, FILE *serialization, FDSet *fds) {
         if (r < 0)
                 return r;
 
-        /* If this is the first boot, and we are in the host system, then preset everything */
-        if (m->first_boot > 0 &&
-            MANAGER_IS_SYSTEM(m) &&
-            !m->test_run_flags) {
-
-                r = unit_file_preset_all(UNIT_FILE_SYSTEM, 0, NULL, UNIT_FILE_PRESET_ENABLE_ONLY, NULL, 0);
-                if (r < 0)
-                        log_full_errno(r == -EEXIST ? LOG_NOTICE : LOG_WARNING, r,
-                                       "Failed to populate /etc with preset unit settings, ignoring: %m");
-                else
-                        log_info("Populated /etc with preset unit settings.");
-        }
-
         lookup_paths_reduce(&m->lookup_paths);
         manager_build_unit_path_cache(m);
 
-- 
2.17.1

