From cd3acb6ae665cebdf0b7eab959f755bb3604ee26 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:50:44 +0100
Subject: [PATCH 14/38] locale-setup: set default locale to a unicode one.

---
 src/core/locale-setup.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/core/locale-setup.c b/src/core/locale-setup.c
index 0c43cf241..afe5d4a60 100644
--- a/src/core/locale-setup.c
+++ b/src/core/locale-setup.c
@@ -82,6 +82,11 @@ int locale_setup(char ***environment) {
                         log_warning_errno(r, "Failed to read /etc/locale.conf: %m");
         }
 
+        /* Nothing set... use some UTF-8 locale */
+        if (r <= 0) {
+                variables[VARIABLE_LANG] = strdup("en_US.UTF-8");
+        }
+
         add = NULL;
         for (i = 0; i < _VARIABLE_LC_MAX; i++) {
                 char *s;
-- 
2.17.1

