From 4d2fc8afd8beba35dd6e66b4304af1d87b553a5e Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Mon, 24 Aug 2015 14:42:53 +0100
Subject: [PATCH 16/38] Ship default services in system unit dir.

---
 units/meson.build | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/units/meson.build b/units/meson.build
index 72560c2ee..75d720b90 100644
--- a/units/meson.build
+++ b/units/meson.build
@@ -65,7 +65,7 @@ units = [
          join_paths(pkgsysconfdir, 'system/multi-user.target.wants/')],
         ['remote-fs-pre.target',                ''],
         ['remote-fs.target',                    '',
-         join_paths(pkgsysconfdir, 'system/multi-user.target.wants/')],
+         'multi-user.target.wants/'],
         ['rescue.target',                       '',
          'runlevel1.target'],
         ['rpcbind.target',                      ''],
@@ -98,7 +98,7 @@ units = [
         ['systemd-journald.socket',             '',
          'sockets.target.wants/'],
         ['systemd-networkd.socket',             'ENABLE_NETWORKD',
-         join_paths(pkgsysconfdir, 'system/sockets.target.wants/')],
+         'sockets.target.wants/'],
         ['systemd-rfkill.socket',               'ENABLE_RFKILL'],
         ['systemd-tmpfiles-clean.timer',        '',
          'timers.target.wants/'],
@@ -173,10 +173,10 @@ in_units = [
         ['systemd-modules-load.service',         'HAVE_KMOD',
          'sysinit.target.wants/'],
         ['systemd-networkd.service',             'ENABLE_NETWORKD',
-         join_paths(pkgsysconfdir, 'system/dbus-org.freedesktop.network1.service') + ' ' +
-         join_paths(pkgsysconfdir, 'system/multi-user.target.wants/')],
+         ('dbus-org.freedesktop.network1.service' + ' ' +
+         'multi-user.target.wants/')],
         ['systemd-networkd-wait-online.service', 'ENABLE_NETWORKD',
-         join_paths(pkgsysconfdir, 'system/network-online.target.wants/')],
+         'network-online.target.wants/'],
         ['systemd-nspawn@.service',              ''],
         ['systemd-poweroff.service',             ''],
         ['systemd-quotacheck.service',           'ENABLE_QUOTACHECK'],
@@ -186,8 +186,8 @@ in_units = [
         ['systemd-remount-fs.service',           '',
          'local-fs.target.wants/'],
         ['systemd-resolved.service',             'ENABLE_RESOLVE',
-         join_paths(pkgsysconfdir, 'system/dbus-org.freedesktop.resolve1.service') + ' ' +
-         join_paths(pkgsysconfdir, 'system/multi-user.target.wants/')],
+         ('dbus-org.freedesktop.resolve1.service' + ' ' +
+         'multi-user.target.wants/')],
         ['systemd-rfkill.service',               'ENABLE_RFKILL'],
         ['systemd-suspend.service',              ''],
         ['systemd-sysctl.service',               '',
@@ -197,7 +197,7 @@ in_units = [
         ['systemd-timedated.service',            'ENABLE_TIMEDATED',
          'dbus-org.freedesktop.timedate1.service'],
         ['systemd-timesyncd.service',            'ENABLE_TIMESYNCD',
-         join_paths(pkgsysconfdir, 'system/sysinit.target.wants/')],
+         'sysinit.target.wants/'],
         ['systemd-tmpfiles-clean.service',       'ENABLE_TMPFILES'],
         ['systemd-tmpfiles-setup-dev.service',   'ENABLE_TMPFILES',
          'sysinit.target.wants/'],
-- 
2.17.1

