From c02c742c26edc93400f7dfb9509c6fa6c3fea4e9 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Sun, 19 Jun 2016 00:51:22 +0000
Subject: [PATCH 16/38] not load iptables

---
 src/core/kmod-setup.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/core/kmod-setup.c b/src/core/kmod-setup.c
index f4488dd692..fdce681ac5 100644
--- a/src/core/kmod-setup.c
+++ b/src/core/kmod-setup.c
@@ -108,10 +108,6 @@ int kmod_setup(void) {
                 /* This should never be a module */
                 { "unix",      "/proc/net/unix",            true,   true,    NULL      },
 
-#if HAVE_LIBIPTC
-                /* netfilter is needed by networkd, nspawn among others, and cannot be autoloaded */
-                { "ip_tables", "/proc/net/ip_tables_names", false,  false,   NULL      },
-#endif
                 /* virtio_rng would be loaded by udev later, but real entropy might be needed very early */
                 { "virtio_rng", NULL,                       false,  false,   has_virtio_rng },
 
-- 
2.36.1

