From e54f8d8a1dfc29462bd07892e619753906371b09 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Thu, 2 Jul 2015 09:43:35 +0100
Subject: [PATCH 20/30] bootchart: drop log_info spam to serial console.

---
 src/bootchart/bootchart.c | 2 --
 src/bootchart/svg.c       | 2 --
 2 files changed, 4 deletions(-)

diff --git a/src/bootchart/bootchart.c b/src/bootchart/bootchart.c
index 0587e9b..5d981d0 100644
--- a/src/bootchart/bootchart.c
+++ b/src/bootchart/bootchart.c
@@ -489,8 +489,6 @@ int main(int argc, char *argv[]) {
                 return EXIT_FAILURE;
         }
 
-        log_info("systemd-bootchart wrote %s\n", output_file);
-
         r = do_journal_append(output_file);
         if (r < 0)
                 return EXIT_FAILURE;
diff --git a/src/bootchart/svg.c b/src/bootchart/svg.c
index af289bf..e743577 100644
--- a/src/bootchart/svg.c
+++ b/src/bootchart/svg.c
@@ -174,8 +174,6 @@ static int svg_title(FILE *of, const char *build, int pscount, double log_start,
                 xsprintf(filename, "/sys/block/%s/device/model", rootbdev);
 
                 r = read_one_line_file(filename, &model);
-                if (r < 0)
-                        log_info("Error reading disk model for %s: %m\n", rootbdev);
         }
 
         /* various utsname parameters */
-- 
2.8.2

