From 39a8f3716a2f684382f6d138f9c9885f5319ceed Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Sun, 19 Jun 2016 00:51:22 +0000
Subject: [PATCH 21/42] not load iptables

---
 src/core/kmod-setup.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/core/kmod-setup.c b/src/core/kmod-setup.c
index a91cfebc67..bfb5377202 100644
--- a/src/core/kmod-setup.c
+++ b/src/core/kmod-setup.c
@@ -87,10 +87,6 @@ int kmod_setup(void) {
                 /* This should never be a module */
                 { "unix",      "/proc/net/unix",            true,   true,    NULL      },
 
-#if HAVE_LIBIPTC
-                /* netfilter is needed by networkd, nspawn among others, and cannot be autoloaded */
-                { "ip_tables", "/proc/net/ip_tables_names", false,  false,   NULL      },
-#endif
                 /* virtio_rng would be loaded by udev later, but real entropy might be needed very early */
                 { "virtio_rng", NULL,                       false,  false,   has_virtio_rng },
         };
-- 
2.20.1

