From 2bea020c6eade846cfe7bf864537792d2f15fd25 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Fri, 27 Nov 2015 16:30:12 +0000
Subject: [PATCH 22/28] tmpfiles: Make /var/cache/ldconfig world readable

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
---
 tmpfiles.d/var.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tmpfiles.d/var.conf b/tmpfiles.d/var.conf
index 03cb0f5..3093884 100644
--- a/tmpfiles.d/var.conf
+++ b/tmpfiles.d/var.conf
@@ -17,6 +17,7 @@ f /var/log/btmp 0600 root utmp -
 
 d /var/cache 0755 - - -
 d /var/cache/locale 0755 - - -
+d /var/cache/ldconfig 0755 - - -
 
 d /var/lib 0755 - - -
 
-- 
2.9.0

