From 0e21da80aa08c54f435c12f39461486a9cb2f322 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Fri, 14 Jul 2017 11:32:54 -0700
Subject: [PATCH 25/45] udev: log also device path.

---
 src/udev/udevd.c | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/src/udev/udevd.c b/src/udev/udevd.c
index ca65474f27..547135b2f0 100644
--- a/src/udev/udevd.c
+++ b/src/udev/udevd.c
@@ -645,8 +645,14 @@ static int event_queue_insert(Manager *manager, sd_device *dev) {
 
         LIST_APPEND(event, manager->events, event);
 
-        log_device_debug(dev, "Device (SEQNUM=%"PRIu64", ACTION=%s) is queued",
-                         seqnum, device_action_to_string(action));
+        if (DEBUG_LOGGING) {
+                const char* devpath;
+                if (sd_device_get_devpath(event->dev, &devpath) < 0)
+                        devpath = NULL;
+
+                log_device_debug(dev, "Device (SEQNUM=%"PRIu64", ACTION=%s, PATH=%s) is queued",
+                                 seqnum, device_action_to_string(action), strnull(devpath));
+        }
 
         return 0;
 }
-- 
2.26.0

