From 26513dc229228d16245a5d1cb6fbcafc00ca19c6 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Sun, 19 Jun 2016 00:51:22 +0000
Subject: [PATCH 26/32] not load iptables

---
 src/core/kmod-setup.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/core/kmod-setup.c b/src/core/kmod-setup.c
index fd1021f70..21a3cc5f2 100644
--- a/src/core/kmod-setup.c
+++ b/src/core/kmod-setup.c
@@ -64,10 +64,6 @@ int kmod_setup(void) {
                 /* this should never be a module */
                 { "unix",      "/proc/net/unix",            true,   true,    NULL      },
 
-#ifdef HAVE_LIBIPTC
-                /* netfilter is needed by networkd, nspawn among others, and cannot be autoloaded */
-                { "ip_tables", "/proc/net/ip_tables_names", false,  false,   NULL      },
-#endif
         };
         struct kmod_ctx *ctx = NULL;
         unsigned int i;
-- 
2.12.2

