From e9ad78d7e2862b8f7f6a73761803f74287d7b2a6 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Fri, 14 Jul 2017 11:32:54 -0700
Subject: [PATCH 28/42] udev: log also device path.

---
 src/udev/udevd.c | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/src/udev/udevd.c b/src/udev/udevd.c
index 0303f36b4f..944c0cd889 100644
--- a/src/udev/udevd.c
+++ b/src/udev/udevd.c
@@ -647,7 +647,15 @@ static int event_queue_insert(Manager *manager, sd_device *dev) {
 
         LIST_APPEND(event, manager->events, event);
 
-        log_device_debug(dev, "Device (SEQNUM=%"PRIu64", ACTION=%s) is queued", seqnum, action);
+        if (DEBUG_LOGGING) {
+                const char* devpath;
+                if (sd_device_get_property_value(dev, "ACTION", &val) < 0)
+                        val = NULL;
+                if (sd_device_get_devpath(event->dev, &devpath) < 0)
+                        devpath = NULL;
+
+                log_device_debug(dev, "Device (SEQNUM=%"PRIu64", ACTION=%s, PATH=%s) is queued", seqnum, strnull(val), strnull(devpath));
+        }
 
         return 0;
 }
-- 
2.20.1

