diff -uprN systemd-229-orig/units/systemd-journal-flush-msft.service.in systemd-229/units/systemd-journal-flush-msft.service.in
--- systemd-229-orig/units/systemd-journal-flush-msft.service.in	1970-01-01 00:00:00.000000000 +0000
+++ systemd-229/units/systemd-journal-flush-msft.service.in	2016-08-15 17:57:39.546455145 +0000
@@ -0,0 +1,22 @@
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Flush Journal to Persistent Storage
+Documentation=man:systemd-journald.service(8) man:journald.conf(5)
+DefaultDependencies=no
+Requires=systemd-journald.service
+After=systemd-journald.service
+After=systemd-remount-fs.service
+RequiresMountsFor=/var/log/journal
+ConditionVirtualization=microsoft
+
+[Service]
+ExecStart=@rootbindir@/journalctl --flush
+Type=oneshot
+RemainAfterExit=yes
+TimeoutSec=90s
diff -uprN systemd-229-orig/units/systemd-journal-flush.service.in systemd-229/units/systemd-journal-flush.service.in
--- systemd-229-orig/units/systemd-journal-flush.service.in	2016-08-15 18:04:37.877736649 +0000
+++ systemd-229/units/systemd-journal-flush.service.in	2016-08-15 18:02:47.952247899 +0000
@@ -14,6 +14,7 @@ After=systemd-journald.service
 After=systemd-remount-fs.service
 Before=systemd-user-sessions.service systemd-tmpfiles-setup.service
 RequiresMountsFor=/var/log/journal
+ConditionVirtualization=!microsoft
 
 [Service]
 ExecStart=@rootbindir@/journalctl --flush
diff -uprN systemd-229-orig/Makefile.am systemd-229/Makefile.am
--- systemd-229-orig/Makefile.am	2016-08-15 18:38:48.055756946 +0000
+++ systemd-229/Makefile.am	2016-08-15 18:41:25.804111742 +0000
@@ -4306,6 +4306,7 @@ dist_systemunit_DATA += \
 nodist_systemunit_DATA += \
 	units/systemd-journald.service \
 	units/systemd-journal-flush.service \
+	units/systemd-journal-flush-msft.service \
 	units/systemd-journal-catalog-update.service
 
 dist_pkgsysconf_DATA += \
@@ -4331,6 +4332,7 @@ SOCKETS_TARGET_WANTS += \
 SYSINIT_TARGET_WANTS += \
 	systemd-journald.service \
 	systemd-journal-flush.service \
+	systemd-journal-flush-msft.service \
 	systemd-journal-catalog-update.service
 
 EXTRA_DIST += \
