From c7b3e0a85852949426482cb3fcc14e5bb8238662 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Fri, 14 Jul 2017 11:34:00 -0700
Subject: [PATCH 36/39] Ensure /var/run is never a directory.

---
 src/core/mount-setup.c | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/src/core/mount-setup.c b/src/core/mount-setup.c
index 1db30c1e5..8dc9b302b 100644
--- a/src/core/mount-setup.c
+++ b/src/core/mount-setup.c
@@ -367,12 +367,21 @@ static int nftw_cb(
 #endif
 
 int mount_setup(bool loaded_policy) {
+	struct stat sb;
+	int ret;
         int r = 0;
 
         r = mount_points_setup(ELEMENTSOF(mount_table), loaded_policy);
         if (r < 0)
                 return r;
 
+	ret = lstat("/var/run", &sb);
+	if (ret == 0 && sb.st_mode & S_IFDIR) {
+		char path[256];
+		sprintf(path, "/var/run.%i", (int)time(NULL));
+		rename("/var/run", path);
+	}
+
 #if defined(HAVE_SELINUX) || defined(HAVE_SMACK)
         /* Nodes in devtmpfs and /run need to be manually updated for
          * the appropriate labels, after mounting. The other virtual
-- 
2.13.3

