From 79729ca2dfb5ca7c5fa7f440a3643670326c2d0e Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Thu, 14 Jun 2018 11:20:27 -0700
Subject: [PATCH] Do not crash if udev hasn't initialized one link yet.

---
 src/network/wait-online/manager.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/network/wait-online/manager.c b/src/network/wait-online/manager.c
index 824f292c3..74437d2b4 100644
--- a/src/network/wait-online/manager.c
+++ b/src/network/wait-online/manager.c
@@ -57,6 +57,7 @@ bool manager_all_configured(Manager *m) {
                 if (!l->state) {
                         log_debug("link %s has not yet been processed by udev",
                                   l->ifname);
+                        continue;
                 }
 
                 if (STR_IN_SET(l->state, "configuring", "pending")) {
-- 
2.19.0

