From 908954bdb31f1d225ac392291df7cb425a676eca Mon Sep 17 00:00:00 2001
From: Arzhan Kinzhalin <arzhan.i.kinzhalin@intel.com>
Date: Tue, 16 Apr 2019 17:48:10 -0700
Subject: [PATCH 40/43] Add dependency on NetworkManager.

Signed-off-by: Arzhan Kinzhalin <arzhan.i.kinzhalin@intel.com>
---
 units/systemd-resolved.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/units/systemd-resolved.service.in b/units/systemd-resolved.service.in
index f73697832c..4d90c5a738 100644
--- a/units/systemd-resolved.service.in
+++ b/units/systemd-resolved.service.in
@@ -14,7 +14,7 @@ Documentation=https://www.freedesktop.org/wiki/Software/systemd/resolved
 Documentation=https://www.freedesktop.org/wiki/Software/systemd/writing-network-configuration-managers
 Documentation=https://www.freedesktop.org/wiki/Software/systemd/writing-resolver-clients
 DefaultDependencies=no
-After=systemd-sysusers.service systemd-networkd.service
+After=systemd-sysusers.service systemd-networkd.service NetworkManager.service
 Before=network.target nss-lookup.target shutdown.target
 Conflicts=shutdown.target
 Wants=nss-lookup.target
-- 
2.24.0

