From 60b98a6d57fec552320f8f1bf635c0f9a04fd904 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Mon, 14 Jan 2019 09:46:32 -0800
Subject: [PATCH 40/44] Disable XZ support in the journal.

---
 src/journal/compress.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/journal/compress.c b/src/journal/compress.c
index 7a79e566b8..e0d85a9ba2 100644
--- a/src/journal/compress.c
+++ b/src/journal/compress.c
@@ -28,6 +28,8 @@
 #include "string-util.h"
 #include "util.h"
 
+#undef HAVE_XZ
+#define HAVE_XZ 0
 #if HAVE_LZ4
 DEFINE_TRIVIAL_CLEANUP_FUNC(LZ4F_compressionContext_t, LZ4F_freeCompressionContext);
 DEFINE_TRIVIAL_CLEANUP_FUNC(LZ4F_decompressionContext_t, LZ4F_freeDecompressionContext);
-- 
2.21.0

