From 5b0649ca25bb9022e8898a6581599c80bbc96e74 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Mon, 14 Jan 2019 09:43:59 -0800
Subject: [PATCH 40/42] Do not enable audit by default in the journal.

---
 src/journal/journald-audit.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/journal/journald-audit.c b/src/journal/journald-audit.c
index accbad4180..193dc64e47 100644
--- a/src/journal/journald-audit.c
+++ b/src/journal/journald-audit.c
@@ -536,10 +536,11 @@ int server_open_audit(Server *s) {
         if (r < 0)
                 return log_error_errno(r, "Failed to add audit fd to event loop: %m");
 
+#if 0
         /* We are listening now, try to enable audit */
         r = enable_audit(s->audit_fd, true);
         if (r < 0)
                 log_warning_errno(r, "Failed to issue audit enable call: %m");
-
+#endif
         return 0;
 }
-- 
2.20.1

