From 538065c02ea9f758f0d500f85c7b62df27ebccc7 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Thu, 10 Oct 2019 14:40:49 -0700
Subject: [PATCH 42/44] Exempt gdm and a few others from default to locked
 account.

---
 src/sysusers/sysusers.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/sysusers/sysusers.c b/src/sysusers/sysusers.c
index f7cc7e0900..0f46038c5f 100644
--- a/src/sysusers/sysusers.c
+++ b/src/sysusers/sysusers.c
@@ -539,6 +539,11 @@ static int write_temporary_shadow(const char *shadow_path, FILE **tmpfile, char
                         .sp_flag = (unsigned long) -1, /* this appears to be what everybody does ... */
                 };
 
+                /* exempt gdm, sddm, lightdm from locking by default */
+                /* https://github.com/systemd/systemd/issues/13522 */
+                if ((i->uid == 311) || (i->uid == 336) || (i->uid == 308))
+                        n.sp_expire = -1;
+
                 r = putspent_sane(&n, shadow);
                 if (r < 0)
                         return r;
-- 
2.26.0

