--- systemd-231/src/libsystemd-network/sd-dhcp-client.c~	2016-07-25 19:49:47.000000000 +0000
+++ systemd-231/src/libsystemd-network/sd-dhcp-client.c	2017-01-24 16:25:44.671406277 +0000
@@ -944,7 +944,7 @@
                 if (client->attempt < 64)
                         client->attempt *= 2;
 
-                next_timeout = time_now + (client->attempt - 1) * USEC_PER_SEC;
+                next_timeout = time_now + (client->attempt - 1) * USEC_PER_SEC / 4;
 
                 break;
 
