--- systemd-233/units/systemd-update-utmp.service.in~	2017-03-01 21:43:06.000000000 +0000
+++ systemd-233/units/systemd-update-utmp.service.in	2017-05-21 16:36:30.188261852 +0000
@@ -12,7 +12,7 @@
 RequiresMountsFor=/var/log/wtmp
 Conflicts=shutdown.target
 After=systemd-remount-fs.service systemd-tmpfiles-setup.service auditd.service
-Before=sysinit.target shutdown.target
+Before=shutdown.target
 
 [Service]
 Type=oneshot
