--- systemd-stable-255.6/tools/elf2efi.py~	2024-05-09 08:07:00.000000000 +0000
+++ systemd-stable-255.6/tools/elf2efi.py	2024-05-09 22:23:24.921753171 +0000
@@ -265,9 +265,9 @@
 
     relro = None
     for elf_seg in elf.iter_segments():
-        if elf_seg["p_type"] == "PT_LOAD" and elf_seg["p_align"] != SECTION_ALIGNMENT:
-            raise BadSectionError(f"ELF segment {elf_seg['p_type']} is not properly aligned"
-                                  f" ({elf_seg['p_align']} != {SECTION_ALIGNMENT})")
+#        if elf_seg["p_type"] == "PT_LOAD" and elf_seg["p_align"] != SECTION_ALIGNMENT:
+#            raise BadSectionError(f"ELF segment {elf_seg['p_type']} is not properly aligned"
+#                                  f" ({elf_seg['p_align']} != {SECTION_ALIGNMENT})")
         if elf_seg["p_type"] == "PT_GNU_RELRO":
             relro = elf_seg
 
