--- systemd-234/src/basic/missing_syscall.h~	2017-07-12 14:01:10.000000000 +0000
+++ systemd-234/src/basic/missing_syscall.h	2018-02-06 15:31:55.424102163 +0000
@@ -59,14 +59,6 @@
 #    endif
 #  endif
 
-static inline int memfd_create(const char *name, unsigned int flags) {
-#  ifdef __NR_memfd_create
-        return syscall(__NR_memfd_create, name, flags);
-#  else
-        errno = ENOSYS;
-        return -1;
-#  endif
-}
 #endif
 
 /* ======================================================================= */
--- systemd-234/src/basic/missing_syscall.h~	2018-02-06 15:31:55.000000000 +0000
+++ systemd-234/src/basic/missing_syscall.h	2018-02-06 15:37:32.169295553 +0000
@@ -31,6 +31,7 @@
 /* ======================================================================= */
 
 #if !HAVE_DECL_MEMFD_CREATE
+#include <sys/mman.h>
 #  ifndef __NR_memfd_create
 #    if defined __x86_64__
 #      define __NR_memfd_create 319
