--- systemd-233/configure.ac~	2017-03-01 21:43:06.000000000 +0000
+++ systemd-233/configure.ac	2017-06-04 12:42:08.953153187 +0000
@@ -228,11 +228,6 @@
                -Wl,--gc-sections])],
         [AC_MSG_RESULT([skipping --gc-sections, optimization not enabled])])
 
-AS_CASE([$CFLAGS], [*-O[[12345sz\ ]]*],
-        [CC_CHECK_FLAGS_APPEND([with_cflags], [CFLAGS], [\
-               -ffunction-sections -fdata-sections])],
-        [AC_MSG_RESULT([skipping -ffunction/data-section, optimization not enabled])])
-
 CC_CHECK_FLAGS_APPEND([with_ldflags], [LDFLAGS], [\
         -Wl,--as-needed \
         -Wl,--no-undefined \
