--- systemd-234/src/resolve/resolved.c.org	2017-11-11 21:52:26.757088407 +0000
+++ systemd-234/src/resolve/resolved.c	2017-11-11 21:52:40.311087962 +0000
@@ -77,6 +77,10 @@
 
         assert_se(sigprocmask_many(SIG_BLOCK, NULL, SIGTERM, SIGINT, SIGUSR1, SIGUSR2, -1) >= 0);
 
+        sd_notify(false,
+                  "READY=1\n"
+                  "STATUS=Processing requests...");
+
         r = manager_new(&m);
         if (r < 0) {
                 log_error_errno(r, "Could not create manager: %m");
@@ -99,10 +103,6 @@
                 goto finish;
         }
 
-        sd_notify(false,
-                  "READY=1\n"
-                  "STATUS=Processing requests...");
-
         r = sd_event_loop(m->event);
         if (r < 0) {
                 log_error_errno(r, "Event loop failed: %m");
