The udevd/udevadm commands are very performance criticial

--- systemd-233/Makefile.am~	2017-03-01 21:43:06.000000000 +0000
+++ systemd-233/Makefile.am	2017-05-21 14:28:43.753882420 +0000
@@ -3947,6 +3947,8 @@
 
 systemd_udevd_SOURCES = \
 	src/udev/udevd.c
+	
+systemd_udevd_CFLAGS = -O3
 
 systemd_udevd_LDADD = \
 	libudev-core.la \
@@ -3968,6 +3970,8 @@
 udevadm_LDADD = \
 	libudev-core.la \
 	libbasic.la
+	
+udevadm_CFLAGS = -O3
 
 # ------------------------------------------------------------------------------
 if ENABLE_HWDB
